packages/salesforce-adapter/src/change_validators/animation_rule_recordtype.ts

```diff
@@ -31,7 +31,7 @@ const invalidRecordTypeError = (instance: InstanceElement): ChangeError => (
     elemID: instance.elemID,
     severity: 'Error',
     message: 'Invalid AnimationRule RecordType',
-    detailedMessage: `In ${instance.elemID.getFullName()}, The RecordTypeId field is missing even though RecordTypeContext requires a RecordTypeId.`,
+    detailedMessage: `In ${instance.elemID.getFullName()}, The RecordTypeId field is missing even though RecordTypeContext requires a RecordTypeId.\nfor more details: https://help.salto.io/en/articles/7793653-deployment-preview-errors`,
```

Comment:
@iperelman small product issue with this text here that we thought of -
In the document there is a guide on how to turn off CV, but to turn them off we require the name of the CV itself which isn't clear from the message.
I suggest something along the following - Does this make sense?
```suggestion
for more details on <Change validator name>: <URL>
```

Reply:
@amitgoldarb yeah, it's probably better.  I'm a little bothered that we do not really share more details about the specific CV there, but rather only about how to turn it off, yet I agree it helps with understanding what CV is in play here + @pgonzaleznetwork is in the process of writing a help page for each of the CVs anyway so soon we should actually have more meaningful details about each CV.